app/parse.py

```diff
@@ -15,8 +40,76 @@ class Product:
     num_of_reviews: int
 
 
+PRODUCT_FIELDS = [field.name for field in fields(Product)]
+
+
+def get_webdriver() -> webdriver.Chrome:
+    chrome_options = Options()
+    chrome_options.add_argument("--headless")
+    chrome_options.add_argument("--disable-gpu")
+    chrome_options.add_argument("--no-sandbox")
+    driver = webdriver.Chrome(options=chrome_options)
+    # driver = webdriver.Chrome()
+    return driver
+
+
+def get_single_product(soup: Tag) -> Product:
+    return Product(
+        title=soup.select_one(".title")["title"],
+        description=soup.select_one(".description").text,
+        price=float(soup.select_one(".price").text.replace("$", "")),
+        rating=int(soup.select_one("p[data-rating]")["data-rating"]),
+        num_of_reviews=int(soup.select_one(".review-count").text.split()[0])
```

Comment:
The code assumes that all elements (title, description, price, rating, num_of_reviews) are present in the HTML. Consider adding checks to handle cases where these elements might be missing or malformed to prevent runtime errors.